src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,33 @@
-export const TodoInfo = () => {};
+import { UserInfo } from '../UserInfo';
+
+interface Todo {
+  id: number;
+  title: string;
+  userId: number;
+  completed: boolean;
+}
+
+interface User {
+  id: number;
+  name: string;
+  email: string;
+}
```

Comment:
please organize your types, because you creating another `Todo` interface in TodoList and `User` interface in `UserInfo`
just use `export` & `import` instead